src/components/Common/MultipleComboBox.tsx

```diff
@@ -0,0 +1,296 @@
+      selectedItem: newSelectedItem,
+    }) {
+      switch (type) {
+        case useCombobox.stateChangeTypes.InputKeyDownEnter:
+        case useCombobox.stateChangeTypes.ItemClick:
+        case useCombobox.stateChangeTypes.InputBlur:
+          if (!newSelectedItem) {
+            return;
+          }
+          addSelectedItem(newSelectedItem);
+          onChangeSelectedItems([...(selectedItems ?? []), newSelectedItem]);
+          setInputValue('');
+
+          break;
+
+        case useCombobox.stateChangeTypes.InputChange:
+          setInputValue(newInputValue);
+
+          break;
+        default:
+          break;
+      }
+    },
+  });
+
+  useLayoutEffect(() => {
+    if (isOpen && refs.reference.current && refs.floating.current) {
+      return autoUpdate(refs.reference.current, refs.floating.current, update);
+    }
+  }, [isOpen, update, refs.floating, refs.reference]);
+
+  return (
+    <>
+      <div className="relative w-full" data-qa="multiple-combobox">
+        <div className="flex w-full flex-col gap-1">
+          {label && <label {...getLabelProps()}>{label}</label>}
+          <div className="relative flex w-full flex-wrap gap-1 rounded border border-primary p-1 focus-within:border-accent-primary">
+            {selectedItems &&
+              selectedItems.map((selectedItemForRender, index) => {
+                return (
+                  <span
+                    className="flex items-center rounded bg-accent-primary-alpha px-3 py-1.5"
+                    key={`selected-item-${getItemLabel(
+                      selectedItemForRender,
+                    )}-${index}`}
+                    {...getSelectedItemProps({
+                      selectedItem: selectedItemForRender,
+                      index,
+                    })}
+                  >
+                    {selectedItemRow ? (
+                      createElement(selectedItemRow, {
+                        item: selectedItemForRender,
+                      })
+                    ) : (
+                      <span className="max-w-[150px] truncate break-all text-xs">
+                        {getItemLabel(selectedItemForRender)}
+                      </span>
+                    )}
+                    <span
+                      data-qa={`unselect-item-${getItemValue(
+                        selectedItemForRender,
+                      )}`}
+                      className="cursor-pointer px-1"
+                      onClick={(e) => {
+                        e.stopPropagation();
+                        removeSelectedItem(selectedItemForRender);
+                      }}
+                    >
+                      <IconX height={12} width={12} />
+                    </span>
+                  </span>
+                );
+              })}
+
+            <input
+              disabled={disabled}
+              placeholder={placeholder || ''}
+              className="bg-transparent px-3 py-1 outline-none placeholder:text-secondary"
```

Comment:
I think we need to limit input by width, also we don't need padding from right:
```suggestion
              className="max-w-[105px] bg-transparent pl-3 py-1 outline-none placeholder:text-secondary"
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/c9b86130-03e3-436a-9491-a990a56f30c4)

Also I thing that padding-left should be small if we have selectedItems:
```jsx
selectedItems.length ? 'pl-1' : 'pl-3'
```
![image](https://github.com/epam/ai-dial-chat/assets/20163971/ffac9427-30a1-4f55-a1ea-6c05b018e0eb)

Reply:
Thank you!
Styles changed as suggested.